.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,6 +22,10 @@
 
 Fixes: #
 
+### Testing
```

Comment:
I think the `Fixes:` part should always be last in the template, but I may be nitpicking here, so feel free to ignore this comment.

Reply:
Nope, makes sense. Will change this :)